tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./node_modules/@sourcegraph/tsconfig/tsconfig.json",
```

Comment:
```suggestion
  "extends": "@sourcegraph/tsconfig",
```

Reply:
I don't think I touched this file (I ran the Sourcegraph script to set up extensions). Unsure, but thanks for catching.